src/main/java/frc/robot/subsystems/BallTransport.java

```diff
@@ -246,6 +247,7 @@ public void periodic() {
 
         seenBallEntry.setBoolean(seenBall);
         laserSwitchEntry.setBoolean(laserSwitch.getAsBoolean());
+        SmartDashboard.putNumber("Color Sensor", colorSensor.getProximity());
```

Comment:
Should probably add "Proximity" to the name

Reply:
Solved